projects/ion/src/lib/icon/svgs/iconsText/cpe.ts

```diff
@@ -0,0 +1,17 @@
+export default `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8.23145 2.17263C11.2502 1.94179 12.922 1.94301 15.7727 2.17296C16.2491 2.21139 16.6317 2.58153 16.6859 3.05638C17.482 10.035 17.9768 13.8503 18.9884 20.44C19.0296 20.7083 18.96 20.9819 18.7955 21.1978C18.631 21.4137 18.3857 21.5535 18.1161 21.585C13.3499 22.1423 10.6482 22.1343 5.88547 21.5852C5.61479 21.554 5.36857 21.4137 5.20374 21.1967C5.03891 20.9798 4.96974 20.7049 5.01224 20.4358C6.07972 13.6758 6.57039 9.87499 7.31359 3.06129C7.36579 2.58277 7.75149 2.20934 8.23145 2.17263ZM9.21096 4.1073C8.55166 10.0968 8.07115 13.8136 7.15035 19.7124C10.8017 20.0934 13.2041 20.098 16.8542 19.7133C15.9801 13.9602 15.4966 10.2346 14.7928 4.10468C12.7794 3.96472 11.3461 3.96463 9.21096 4.1073Z" fill="#282B33"/>
```

Comment:
In order for the color of this icon to be controlled by the ion-icon component, please remove the fill attributes from the SVG code.

```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M8.23145 2.17263C11.2502 1.94179 12.922 1.94301 15.7727 2.17296C16.2491 2.21139 16.6317 2.58153 16.6859 3.05638C17.482 10.035 17.9768 13.8503 18.9884 20.44C19.0296 20.7083 18.96 20.9819 18.7955 21.1978C18.631 21.4137 18.3857 21.5535 18.1161 21.585C13.3499 22.1423 10.6482 22.1343 5.88547 21.5852C5.61479 21.554 5.36857 21.4137 5.20374 21.1967C5.03891 20.9798 4.96974 20.7049 5.01224 20.4358C6.07972 13.6758 6.57039 9.87499 7.31359 3.06129C7.36579 2.58277 7.75149 2.20934 8.23145 2.17263ZM9.21096 4.1073C8.55166 10.0968 8.07115 13.8136 7.15035 19.7124C10.8017 20.0934 13.2041 20.098 16.8542 19.7133C15.9801 13.9602 15.4966 10.2346 14.7928 4.10468C12.7794 3.96472 11.3461 3.96463 9.21096 4.1073Z"/>
```